packages/itwinui-react/package.json

```diff
@@ -104,6 +104,7 @@
     "@floating-ui/react": "^0.26.18",
     "@itwin/itwinui-illustrations-react": "^2.1.0",
     "@swc/helpers": "^0.5.11",
+    "@tanstack/react-virtual": "^3.5.0",
```

Comment:
lets use latest (3.8.2)

https://github.com/TanStack/virtual/releases

Reply:
Updated the package in this [commit](https://github.com/iTwin/iTwinUI/pull/2061/commits/4975651c8ba1606b3ff0d3effdbc292ca9ad48a1). There was a change in the `VirtualItem` type that meant that all instances of it across the branches had to be changed to `VirtualItem<Element>`. I did this in this [commit](https://github.com/iTwin/iTwinUI/pull/2092/commits/3ebe316f8cabcacf49030ad35de19d23989c8cfc) for `Table` and this [commit](https://github.com/iTwin/iTwinUI/pull/2091/commits/afcbd2df67e43f34dac0153688dcbb3447bb211c) for `Tree`.